unfoldingWord/translationCore
# Refactor/klappy/containers/#1774

#### This pull request addresses:

Refactor containers.


#### How to test this pull request:

Load app, check modal, load project/tools, swap tools and all should work.


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/unfoldingword-dev/translationcore/1798)
